addons/im_livechat/static/src/models/public_livechat_window.js

```diff
@@ -27,6 +26,18 @@ registerModel({
             this.legacyChatWindow.destroy();
         },
     },
+    recordMethods: {
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeInputPlaceholder() {
+            if (this.messaging.livechatButtonView.inputPlaceholder) {
+                return this.messaging.livechatButtonView.inputPlaceholder;
+            }
+            return this.env._t("Say something");
```

Comment:
where does this come from?

Reply:
I forgot to remove where it was used. You should see the negative diff that put this "Say something" as default.